dfastmi/gui/dialog_view.py

```diff
@@ -933,31 +955,29 @@ def _menu_open_manual(self):
         ---------
         None
         """
-        subprocess.Popen(self._view_model.manual_filename, shell=True)
+        os.startfile(self._view_model.manual_filename)
```

Comment:
good improvement.
**Windows Only**: Use os.startfile(self._view_model.manual_filename) for simplicity and safety when you just need to open a file with its default application.

Reply:
Discussed with Marlon, can be closed.